src/app/services/task.service.ts

```diff
@@ -2,24 +2,23 @@ import {Injectable} from '@angular/core';
 import {HttpClient} from "@angular/common/http";
 import {TaskModel} from "../shared/task.model";
 import {BehaviorSubject, first, Observable} from "rxjs";
-import {JsonStructureModel} from "../shared/json-structure.model";
 
 @Injectable({
   providedIn: 'root'
 })
 export class TaskService {
   taskSubject = new BehaviorSubject<TaskModel[]>([]);
-  tasks$ = this.taskSubject.asObservable();
-  FILE_PATH: string = '../../../assets/dummy-tasks.json';
+  tasks$: Observable<TaskModel[]> = this.taskSubject.asObservable();
+  TASK_ENDPOINT: string = 'http://localhost:5050/tasks';
```

Comment:
instead of declaring these variables here, you might need to reuse them somewhere else , for example, base url:
http://localhost:5050, it would be better to move them all into a constants.ts file, also try to adjust the app to run in SSL mode (https instead of http)